integrations/castle/lib/index.js

```diff
@@ -5,39 +5,44 @@
  */
 
 var integration = require('@segment/analytics.js-integration');
+var objCase = require('obj-case');
+var newDate = require('new-date');
+var isoDate = require('@segment/isodate');
+var toIsoString = require('@segment/to-iso-string');
+var isObject = require('isobject');
 
 /**
  * Expose `Castle` integration.
  */
 
-var Castle = (module.exports = integration('Castle')
+var CastleIntegration = (module.exports = integration('Castle')
+  .global('CastleSegment')
+  .global('casData')
   .option('publishableKey', '')
-  .option('autoPageview', false)
```

Comment:
This functionality is no longer supported in the new version?

Reply:
yes it's not supported anymore. I'm not sure, Should make a dummy placeholder option just to cover this ?

Reply:
let's keep it removed :)